### DISCUSSION TITLE
Progress Bar

### DISCUSSION THREAD

**Discussion by KaapaTeelb** [akai-katto/dandere2x#188] [Ideas]
It would be nice to have a progress bar or atleast ETA for the video in render.

**Comment by KaapaTeelb** [comment]
Oop, my bad;

My half an hour video was preparing for like 40 minutes;
Then the progress appeared.

My bad.

But, it would be nice to make ETAs for preparation then.

**Reply by akai-katto** [reply]
This would be good - having to re-encode videos removed a lot of dandere2x bugs so I think it's staying unfortunately, but having an ETA for pre-processing would help

**Reply by akai-katto** [reply]
Submitting an issue for me to track this would help, if you feel the desire to do that
